src/ToggleButton/Option.js

```diff
@@ -0,0 +1,253 @@
+
+/**
+ * Represents an option within a Toggle Button
+ *
+ */
+
+export const styleSheet = createStyleSheet('MuiOption', {
+  root: {
+    backgroundColor: 'transparent',
+    borderRadius: 0,
+    fontWeight: 600,
+    fontSize: 14,
+    float: 'left',
+    padding: '0px 12px',
+    textTransform: 'uppercase',
+    // borderLeft: props.optionStyle.borderLeft? props.optionStyle.borderLeft : 'none',
+    // borderRight: props.optionStyle.borderRight? props.optionStyle.borderRight : 'none',
+    height: 36,
+  },
+  button: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    color: fade(common.black, 0.3),
+  },
+  dropDownButton: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    float: 'left', // TODO: Move this down to where the dropdown is being generated
+    color: fade(common.black, 0.3),
+  },
+  textSelected: {
+    color: fade(common.black, 0.54),
+  },
+  backgroundSelected: {
+    backgroundColor: fade(common.black, 0.3),
+  },
+  iconAndText: {
+    height: 58,
+  },
+  divided: {
+    borderLeft: `0.25px solid ${grey[500]}`,
+  },
+});
+
+class Option extends Component {
+  static muiName = 'Option';
+
+  static defaultProps = {
+    selected: false,
+  };
+
+  componentWillMount() {
+    if (this.props.children) {
+      this.setState({
+        value: null,
+        anchorEl: undefined,
+        open: false,
+      });
+    }
+  }
+
+  handleDropDownClick = item => {
+    const newValue = this.props.value === item.props.value ? null : item.props.value;
+    this.setState({
+      value: newValue,
+      open: false,
+    });
+    if (this.props.onClick) {
+      const props = Object.assign({}, this.props);
+      props.value = newValue;
+      this.props.onClick(props);
+    }
+  };
+
+  handleOptionClick = () => {
+    if (this.props.onClick && !this.props.children) {
+      this.props.onClick(this.props);
+    }
+  };
+
+  handleClick = event => {
+    this.setState({ open: true, anchorEl: event.currentTarget });
+  };
+
+  handleRequestClose = () => {
+    this.setState({ open: false });
+  };
+
+  render() {
+    const { icon, label, classes, divider, noBackground, selected, children } = this.props;
+
+    const buttonProps = {
+      className: classNames(classes.root, {
+        [classes.iconAndText]: icon && label,
+        [classes.backgroundSelected]: selected && !noBackground,
+        [classes.divided]: divider,
+      }),
+      onClick: this.handleOptionClick,
+      centerRipple: true,
+    };
+    const rootProps = {
+      className: classNames(classes.button, {
+        [classes.textSelected]: selected,
+      }),
+    };
+    const dropDownProps = {
+      className: classNames(classes.dropDownButton, {
+        [classes.textSelected]: selected,
+      }),
+    };
+
+    let optionButton;
+    let option;
+    // If there are children available, make it a Dropdown Menu.
```

Comment:
It doesn't seem to be a segmented drop down button within the specification therefore I would be inclined to agree with your previous statement about it possibly just being there for appearance. 

All tings considered, what would the accepted functionality be?
 
Personally I agree with the dropdown being separated if the current implementation isn't acceptable.